lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers.go

```diff
@@ -210,6 +210,17 @@ func listSubscribersV2Handler(c echo.Context) error {
 
 	var pageSize uint64 = 0
 	var err error
+
+	// get total number of subscribers
+	loadCriteria := configurator.EntityLoadCriteria{}
+	count, err := configurator.CountAllEntitiesOfType(
```

Comment:
Let's not make a configurator call here, since we may be making an early return if ParamMSISDN or ParamIP are set. Move the configurator call below those